### PR TITLE
Add GSoC idea for Chain Core

### PR DESCRIPTION
**Pull request by tonychain**

### CODE REVIEW

File: ideas.html [hunk 1]
```diff
@@ -95,7 +95,7 @@ <h1 class="page-header">Google Summer of Code</h1>
 
             <p>Interledger has specific service requirements from any payment network/ledger that becomes part of the network. The most common way to ILP-enable a ledger is to write a plugin/adaptor for that ledger that conforms to the Interledger specifications</p>
 
-            <p>The community has been particulalry excited about integrating the Bitcoin Lightning and Ethereum Raden netwokrs into the Interledger so experience with eithr of these would be a big bonus.</p>
+            <p>The community has been particulalry excited about integrating the Bitcoin Lightning and Ethereum Raden netwokrs into the Interledger so experience with either of these would be a big bonus.</p>
```
Comment (tonychain) [inline ideas.html]: Fixed a typo here, FYI

Comment (adrianhopebailie) [inline ideas.html]: Thanks

**Comment by emschwartz** [comment]
Unfortunately the GSoC application got turned down...

**Comment by adrianhopebailie** [comment]
> Unfortunately the GSoC application got turned down...

Damn! Well we should keep this page going anyway as a dynamic todo list for new contributors

**Comment by tonychain** [comment]
Oh well. Maybe next year!

**Comment by adrianhopebailie** [comment]
> Oh well. Maybe next year!

Hope most of this is done before then

**Pull request merged**